concordium-node/src/consensus_ffi/helpers.rs

```diff
@@ -249,6 +261,16 @@ impl ConsensusFfiResponse {
             Success | PendingFinalization | Asynchronous => true,
         }
     }
+
+    /// Get the label. This is used when updating metrics of the prometheus
+    /// exporter.
+    pub fn as_label(&self) -> &str {
+        match self {
+            ConsensusFfiResponse::Success => "valid",
+            ConsensusFfiResponse::DuplicateEntry => "duplicate",
```

Comment:
We should use the `is_success`  for `valid`. Not just the `Success` variant.

Duplicate is what we want here I think though since it measures purely duplicate messages.